application/src/main/java/org/exoplatform/chat/listener/UpdateUserEventListener.java

```diff
@@ -12,10 +14,25 @@
 
   public void postSave(User user, boolean isNew) throws Exception {
     try {
-      String currentUserId = ConversationState.getCurrent().getIdentity().getUserId();
+      String dbName = "";
+      String prefixDB = PropertyManager.getProperty(PropertyManager.PROPERTY_DB_NAME);
+      ConversationState currentState = ConversationState.getCurrent();
+      if (currentState != null) {
+        dbName = (String) currentState.getAttribute("currentTenant");
+      }
+      if (StringUtils.isEmpty(dbName)) {
+        dbName = prefixDB;
+      } else {
+        StringBuilder sb = new StringBuilder()
+                                      .append(prefixDB)
+                                      .append("_")
+                                      .append(dbName);
+        dbName = sb.toString();
+      }
```

Comment:
Please factorize this method to retrieve the current database name in order to avoid duplicating this code.